_data/153.yml

```diff
@@ -13,94 +13,94 @@ downloads:
   - name: aarch64
     types:
     - name: offline_image
       desc: offline_desc
-      primary_link: /distribution/leap/15.3/iso/openSUSE-Leap-15.3-2-DVD-aarch64-Current.iso
+      primary_link: /distribution/leap/15.3/iso/openSUSE-Leap-15.3-DVD-aarch64-Current.iso
       links:
       - name: metalink
-        url: /distribution/leap/15.3/iso/openSUSE-Leap-15.3-2-DVD-aarch64-Current.iso.meta4
+        url: /distribution/leap/15.3/iso/openSUSE-Leap-15.3-DVD-aarch64-Current.iso.meta4
       - name: pick_mirror
-        url: /distribution/leap/15.3/iso/openSUSE-Leap-15.3-2-DVD-aarch64-Current.iso?mirrorlist
+        url: /distribution/leap/15.3/iso/openSUSE-Leap-15.3-DVD-aarch64-Current.iso?mirrorlist
       - name: checksum
-        url: /distribution/leap/15.3/iso/openSUSE-Leap-15.3-2-DVD-aarch64-Current.iso.sha256
+        url: /distribution/leap/15.3/iso/openSUSE-Leap-15.3-DVD-aarch64-Current.iso.sha256
     - name: network_image
       desc: network_desc
-      primary_link: /distribution/leap/15.3/iso/openSUSE-Leap-15.3-2-NET-aarch64-Media.iso
+      primary_link: /distribution/leap/15.3/iso/openSUSE-Leap-15.3-NET-aarch64-Current.iso
       links:
       - name: metalink
-        url: /distribution/leap/15.3/iso/openSUSE-Leap-15.3-2-NET-aarch64-Media.iso.meta4
+        url: /distribution/leap/15.3/iso/openSUSE-Leap-15.3-Current-aarch64-Media.iso.meta4
       - name: pick_mirror
-        url: /distribution/leap/15.3/iso/openSUSE-Leap-15.3-2-NET-aarch64-Media.iso?mirrorlist
+        url: /distribution/leap/15.3/iso/openSUSE-Leap-15.3-Current-aarch64-Media.iso?mirrorlist
       - name: checksum
-        url: /distribution/leap/15.3/iso/openSUSE-Leap-15.3-2-NET-aarch64-Media.iso.sha256
+        url: /distribution/leap/15.3/iso/openSUSE-Leap-15.3-Current-aarch64-Media.iso.sha256
   - name: ppc64le
     types:
     - name: offline_image
       desc: offline_desc
-      primary_link: /distribution/leap/15.3/iso/openSUSE-Leap-15.3-2-DVD-ppc64le-Current.iso
+      primary_link: /distribution/leap/15.3/iso/openSUSE-Leap-15.3-DVD-ppc64le-Current.iso
       links:
       - name: metalink
-        url: /distribution/leap/15.3/iso/openSUSE-Leap-15.3-2-DVD-ppc64le-Current.iso.meta4
+        url: /distribution/leap/15.3/iso/openSUSE-Leap-15.3-DVD-ppc64le-Current.iso.meta4
       - name: pick_mirror
-        url: /distribution/leap/15.3/iso/openSUSE-Leap-15.3-2-DVD-ppc64le-Current.iso?mirrorlist
+        url: /distribution/leap/15.3/iso/openSUSE-Leap-15.3-DVD-ppc64le-Current.iso?mirrorlist
       - name: checksum
-        url: /distribution/leap/15.3/iso/openSUSE-Leap-15.3-2-DVD-ppc64le-Current.iso.sha256
+        url: /distribution/leap/15.3/iso/openSUSE-Leap-15.3-DVD-ppc64le-Current.iso.sha256
     - name: network_image
       desc: network_desc
-      primary_link: /distribution/leap/15.3/iso/openSUSE-Leap-15.3-2-NET-ppc64le-Media.iso
+      primary_link: /distribution/leap/15.3/iso/openSUSE-Leap-15.3-Current-ppc64le-Current.iso
```

Comment:
That was a `Current` too much ;-)

Fixed with f654ca892b7e5d2111ab4267751edc8c9540fed2, will be deployed in 30 minutes.